CHANGELOG.unreleased.md

```diff
@@ -9,6 +9,7 @@
 
 - [LDAP] Prevent LDAP-provided groups from being edited from XO [#1884](https://github.com/vatesfr/xen-orchestra/issues/1884) (PR [#5351](https://github.com/vatesfr/xen-orchestra/pull/5351))
 - [Licensing] Allow Free and Starter users to copy VMs and create a VM from snapshot on the same pool [#4890](https://github.com/vatesfr/xen-orchestra/issues/4890) (PR [5333](https://github.com/vatesfr/xen-orchestra/pull/5333))
+- [Trial edition] Trial edition users have a new banner that inform about trial end [#558](https://gitlab.com/vates/company/-/issues/558) (PR [#5374](https://github.com/vatesfr/xen-orchestra/pull/5374))
```

Comment:
```suggestion
- [Licensing] Add trial end information banner [#558](https://gitlab.com/vates/company/-/issues/558) (PR [#5374](https://github.com/vatesfr/xen-orchestra/pull/5374))
```

Reply:
I don't think the GitLab link is useful here.